meson.build

```diff
@@ -28,20 +28,20 @@ if syslibdir == ''
     syslibdir = '/' / get_option('libdir')
 endif
 
-direct = get_option('direct') == 'enabled'
-sgx = get_option('sgx') == 'enabled'
-skeleton = get_option('skeleton') == 'enabled'
+direct = get_option('direct').enabled()
+sgx = get_option('sgx').enabled()
+skeleton = get_option('skeleton').enabled()
 
-dcap = get_option('dcap') == 'enabled'
-ubsan = get_option('ubsan') == 'enabled'
-asan = get_option('asan') == 'enabled'
-vtune = get_option('vtune') == 'enabled'
+dcap = get_option('dcap').enabled()
+ubsan = get_option('ubsan').enabled()
+asan = get_option('asan').enabled()
+vtune = get_option('vtune').enabled()
 
-enable_libgomp = get_option('libgomp') == 'enabled'
-enable_tests = get_option('tests') == 'enabled'
+enable_libgomp = get_option('libgomp').enabled()
+enable_tests = get_option('tests').enabled()
 
-enable_glibc = get_option('glibc') == 'enabled'
-enable_musl = get_option('musl') == 'enabled'
+enable_glibc = get_option('glibc').enabled()
+enable_musl = get_option('musl').enabled()
```

Comment:
What's the expected behavior if someone configures these options with the value e.g. `-Dmusl=auto`?